tests/tests/installation.spec.js

```diff
@@ -63,67 +65,106 @@ test('00 disable namespace filter', async({ page }) => {
   await expect(page.getByText('Namespace: cattle-system')).toBeVisible()
 });
 
+test('00 enable extension developer features', async({ page }) => {
+  if (ORIGIN != 'source') test.skip(true, "Skip: Use developer load only when testing source code")
+  await page.goto('/dashboard/prefs')
+  await expect(page.getByRole('heading', { name: 'Advanced Features' })).toBeVisible()
+  await page.getByRole('checkbox', { name: 'Enable Extension developer features' }).check();
+});
+
 
 // ==================================================================================================
 // Installation
 
 test('01 enable extension support', async({ page }) => {
-  if (ORIGIN == 'source') test.skip(true, 'Loading extension from yarn dev')
-
   // menu -> configuration -> extensions
   await page.goto('/dashboard/c/local/uiplugins');
+  await expect(page.getByRole('heading', { name: 'Extension support is not enabled' })).toBeVisible()
 
-  // click Enable button
+  // Enable extensions
   await page.getByRole('button', { name: 'Enable' }).click();
+  // don't add released extension repository
+  if (ORIGIN != 'released') {
+    await page.locator('label.checkbox-container')
+      .filter({has: page.getByText('Add the Rancher Extension Repository')})
+      .locator('span.checkbox-custom').uncheck()
+  }
   await page.getByRole('button', { name: 'OK' }).click();
 
-  // wait for extension list
+  // Wait for extensions to be enabled
   try {
-    await expect(page.getByRole('tab', { name: 'Installed' })).toBeVisible({timeout: 30_000});
+    await expect(page.getByRole('tab', { name: 'Installed' })).toBeVisible({timeout: 60_000});
   } catch (e) {
-    console.log('Reload - Not showing list of extensions')
+    console.log('Reload - Not showing installed extensions tab')
     await page.reload();
     await expect(page.getByRole('tab', { name: 'Installed' })).toBeVisible();
   }
 
-  // wait for extensions
-  await page.getByRole('tab', { name: 'Available' }).click()
-  await page.waitForTimeout(5000)
-  await page.reload()
-  await expect(page.locator('.plugin', {hasText: 'Kubewarden'} )).toBeVisible();
-});
+  // Wait for default list of extensions
+  await expect(page.locator('.plugin', {hasText: 'Virtualization Manager'} )).toBeVisible();
```

Comment:
This might fail once [this pr](https://github.com/kubewarden/ui/pull/323/files) is merged as I'm removing the unnecessary mention of harvester manger leftover from a previous dashboard bug.

Reply:
I think this is not relates, I am checking extensions were enabled by waiting for preinstalled Virt Manager

![Screenshot from 2023-04-20 13-28-07](https://user-images.githubusercontent.com/538604/233353445-2d86b00d-9a68-47de-b731-d67ffcc1009d.png)